SYSTEM_REQUIREMENTS.md

```diff
@@ -54,6 +55,12 @@ This document provides details about hardware, operating system, and software pr
 - Systems with Android* operating systems
   - Android* 9
 
+### Community-Supported Platforms
+- MinGW*
+- FreeBSD*
+- Microsoft* Windows* on ARM*/ARM64*
+- macOS* on ARM*/ARM64*
+
```

Comment:
Suggest:
```
- macOS* on ARM64*
```

because macOS support only 64-bit arm.

Reply:
@alexandraepan Please, apply this suggestion since it is relevant.

Reply:
Thanks! Have updated the PR @phprus @omalyshe